test/normalize2/elems-mods.test.js

```diff
@@ -65,3 +66,18 @@ test('should support mods in elems and block', t => {
         { entity: { block: 'block', elem: 'elem', modName: 'm2', modVal: 'v2' }, tech: undefined }
     ]);
 });
+
+test('should support block mods with `elems` field without block', t => {
+    const decl = [
+        {
+            elems: ['close'],
+            mods: { theme: 'protect' }
+        }
+    ];
+
+    t.deepEqual(normalize(decl), [
+        { entity: { block: null }, tech: undefined },
```

Comment:
guys why it normalize to `{ entity: { block: null }, tech: undefined },` is it really usefull ?
who needs empty block with empty tech ?